src/main/src/main/java/org/geoserver/logging/GeoServerXMLConfiguration.java

```diff
@@ -120,61 +121,133 @@ protected void stripConsoleStout(Node node) {
         }
     }
 
+    /**
+     * Check node name (xml element and attribute names are not case sensitive, and have not been
+     * corrected yet).
+     *
+     * @param node config node
+     * @param type node type
+     * @return node name matches type
+     */
     protected boolean isType(Node node, String type) {
-        return node != null && node.getName() != null && node.getName().equals(type);
+        return node != null && node.getName() != null && node.getName().equalsIgnoreCase(type);
+    }
+    /**
+     * Check node has attribute (xml element and attribute names are not case sensitive, and have
+     * not been corrected yet).
+     *
+     * <p>Use this to preform a case-insenstive lookup of attribute value: {@code
+     * node.getAttribute().get(attributeName(node,"fileName"))}.
+     *
+     * @param node Config node
+     * @param name Attribute name
+     * @return node attribute matching name, or null if not found
+     */
+    static final String attribute(Node node, String name) {
```

Comment:
Seems to be used only once in a test, in a weird way... I'd suggest removing and using attributeGet instead (see below).